demo/berlin.R

```diff
@@ -197,28 +197,41 @@ tibble(x = 1:5) %>%
 
 cdm_nycflights13()
 
-# Filtering on a table returns a dm object
-cdm_nycflights13() %>%
-  cdm_filter(airlines, carrier == "AA")
+# Filtering on a table returns a dm object with the filter condition(s) stored
+(dm_nyc_filtered <-
```

Comment:
This does its job, but I'm not a big fan of using `()` to force printing. I'll review and change later.